### PR TITLE
Adding notices

### PR DESCRIPTION
**Pull request by datamattsson**
Signed-off-by: Michael Mattsson <michael.mattsson@gmail.com>

### CODE REVIEW

File: docs/legal/notices/index.md [hunk 1]
```diff
@@ -0,0 +1,1092 @@
+Attributions for third party components.
+
+[TOC]
```
Comment (rkumpf) [inline docs/legal/notices/index.md:3]: Any reason to include this?  I wonder what it looks like if the only entry is at level 5.

Comment (datamattsson) [reply docs/legal/notices/index.md:3]: The entry is now on level 1.

**Review by rkumpf — APPROVED** [review]

File: docs/legal/notices/index.md [hunk 2]
```diff
@@ -0,0 +1,1092 @@
...
+
+You may add Your own copyright statement to Your modifications and may provide additional or different license terms and conditions for use, reproduction, or distribution of Your modifications, or for any such Derivative Works as a whole, provided Your use, reproduction, and distribution of the Work otherwise complies with the conditions stated in this License.
+5. Submission of Contributions. Unless You explicitly state otherwise, any Contribution intentionally submitted for inclusion in the Work by You to the Licensor shall be under the terms and conditions of this License, without any additional terms or conditions. Notwithstanding the above, nothing herein shall supersede or modify the terms of any separate license agreement you may have executed with Licensor regarding such Contributions.
```
Comment (rkumpf) [inline docs/legal/notices/index.md:694]: I would add a blank line above, if it were me.  :)

File: docs/legal/notices/index.md [hunk 3]
```diff
@@ -0,0 +1,1092 @@
...
+
+"Contribution" shall mean any work of authorship, including the original version of the Work and any modifications or additions to that Work or Derivative Works thereof, that is intentionally submitted to Licensor for inclusion in the Work by the copyright owner or by an individual or Legal Entity authorized to submit on behalf of the copyright owner. For the purposes of this definition, "submitted" means any form of electronic, verbal, or written communication sent to the Licensor or its representatives, including but not limited to communication on electronic mailing lists, source code control systems, and issue tracking systems that are managed by, or on behalf of, the Licensor for the purpose of discussing and improving the Work, but excluding communication that is conspicuously marked or otherwise designated in writing by the copyright owner as "Not a Contribution."
```
Comment (rkumpf) [inline docs/legal/notices/index.md:678]: I think this ends up without word wrapping, so you have to pan way to the right to see it all.  Anything we can do about it?

Comment (datamattsson) [reply docs/legal/notices/index.md]: fold -w 80 -s

**Review by c-snell — APPROVED** [review]
lgtm

**Pull request merged**